src/components/MenuBars/MenuBars.scss

```diff
@@ -60,77 +203,10 @@ $box-shadow: 0px 10px 20px #00000026;
   height: 100%;
 }
 
-.menu-bars__switch {
-  background-color: $background-color-light;
-  box-shadow: $box-shadow;
-  height: $action-bar-width;
-  width: $action-bar-width;
-  padding: 0;
-  border: 0;
-  border-radius: 100%;
-  transition: background-color 0.25s ease-in-out, filter 0.08s ease-out;
-  // 206px is the width of the horizontal action bar
-  margin-left: 206px + $margin--small;
-  position: relative;
-  cursor: pointer;
-  display: none;
-}
-.menu-bars__switch:hover {
-  filter: brightness(1.4);
-}
-.switch__icon {
-  position: absolute;
-  top: 0;
-  left: 0;
-  transform: rotate(0deg);
-  transition: opacity 0.5s, visibility 0.5s, transform 1s;
-  color: $color-black;
-}
 .menu-animation {
```

Comment:
Is this still needed? If not, there's also some dead code in `MenuBars.tsx`, such as the `animate` state, the `handleAnimate` function and some imports